compiler_setup/nvidia_acc.sh

```diff
@@ -0,0 +1,48 @@
+# Build settings for the NVIDIA compiler with OpenACC
+# ===================================================
+# Fortran compiler
+F90=nvfortran
+# C compiler
+CC=nvcc
+CFLAGS="-g"
+# Fortran compiler flags
+F90FLAGS="-O3 -Minfo=all"
+# Debugging options
+#F90FLAGS"+=" -fcheck=all -fbacktrace -ffpe-trap=invalid -g -O0"
+# -Mcuda is for CUDA Fortran
+# nordc - do not link to routines compiled for device (ensure
+# kernel code is in-lined in loops)
+# cc = compute capability
+# Registers are shared by threads in an SMP. The more registers a kernel
+# uses, the fewer threads it can support. This parameter can be tuned and
+# shoul be a multiple of 8.
+# -Mcuda is required to build CUDA Fortran
+# For Quadro K600
+#F90FLAGS+=" -acc -ta=tesla:cc30,nordc -Mcuda=cc30,nordc"
+# For Tesla K20c
+#F90FLAGS+=" -acc -ta=tesla,cc35,maxregcount:80,nordc -Mcuda=cc35,maxregcount:80,nordc"
+# V100 with managed memory
+F90FLAGS+=" -acc=gpu -gpu=cc70,managed"
+# Linker flags
+# For Quadro K600
+#LDFLAGS+=" -acc -ta=tesla,cc30 -Mcuda=cc30,nordc"
+# For Tesla K20c
+#LDFLAGS="-acc -ta=nvidia,cc35 -Mcuda=cc35,nordc"
+# V100 with managed memory
+LDFLAGS="-acc=gpu -gpu=cc70,managed"
+# Location of various CUDA maths libraries
+LDFLAGS+=" -Mcuda -L${CUDA_MATH_DIR}/lib64  -lnvToolsExt"
+# Flags to use when compiling with OpenMP support
+OMPFLAGS="-mp"
+# Command to use to create archive of object files
+AR=ar
+# Location of PSyclone NVIDIA profiling library (used when adding
+# profiling to the nemo/tracer_advection benchmark).
+PSYCLONE_NVIDIA_LIB_DIR=/home/aporter/PSyclone/lib/profiling/nvidia
```

Comment:
Hard-coded path. Perhaps add a $NVIDIA_HOME and tell users to set it appropriately in a comment (and the readme).

Reply:
I've added the need to set this variable in the two appropriate READMEs. It would be better if this repo had PSyclone as a submodule (it already has Kokkos) and then it could all be built automatically (and somewhat tested using the CI).

Reply:
I agree with this since specific version of PSycloneBench are tied to the PSyclone master at that time. Not always happens but if you go back to a previous commit but keep your current master PSyclone some scripts will not work at all. Keeping it as a submodule will specify exactly what worked at that time. (The con is that this is a submodule that we will have to keep updating pretty much every single time)

Reply:
Yes, that makes sense to me too. A separate issue though.